packages/sdk/src/types/runtime/actor.ts

```diff
@@ -384,6 +389,44 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
         this._text.copy(text);
     }
 
+    /**
+     * Instruct the actor to face another object, or stop facing an object.
+     * @param actorOrActorId The Actor or id of the actor to face.
+     * @param lookAtMode (Optional) How to face the target. @see LookUpMode.
+     * @param backward (Optional) If true, actor faces away from target rather than toward.
+     */
+    public enableLookAt(actorOrActorId: Actor | string, mode?: LookAtMode, backward?: boolean) {
+        // Resolve the actorId value.
+        let actorId = ZeroGuid;
+        if (typeof (actorOrActorId) === 'object' && actorOrActorId.id !== undefined) {
+            actorId = actorOrActorId.id;
+        } else if (typeof (actorOrActorId) === 'string') {
+            actorId = actorOrActorId;
+        }
+        // Allocate component if necessary.
+        if (!this._lookAt) {
+            this._lookAt = new LookAt();
+            // Set up observed properties in the next tick, giving us a chance to initialize the values first (below).

```

Comment:
>// Set up observed properties in the next tick, giving us a chance to initialize the values first (below). [](start = 12, length = 106)

I think duplicating the copy call for the initialization wouldn't be as bad as delaying the observation. That delay seems really strange.

Reply:
Agreed. I tried this a few different ways. Let me update to a better one. Once we're settled on a good pattern, I'll update the other enable* calls to match.

Reply:
Removed call to nextTick.

---
In reply to: [261443428](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/225#discussion_r261443428) [](ancestors = 261443428)